pkg/importer/importer.go

```diff
@@ -124,6 +124,12 @@ type Properties struct {
 	// For example, this is "foursquare" for "swarm", so "swarm" shows
 	// in the UI and URLs, but it's "foursquare" in permanodes.
 	PermanodeImporterType string
+
+	// HasSomePaidAPI specifies whether this service has paid API(s)
+	// that we might need to use to extract data.
```

Comment:
has part of its API requiring payment or quota for optional bits of data (such as Swarm venue photos).